setup.py

```diff
@@ -54,18 +51,13 @@
                          for d, folders, files in os.walk(resource_dir)]
 
 
-setup(
-    name='data-profiler',
+DESCRIPTION = "What is in your data? Detect schema, statistics and entities in almost any file."
 
-    # Versions should comply with PEP440.  For a discussion on single-sourcing
-    # the version across setup.py and the project code, see
-    # https://packaging.python.org/en/latest/single_source_version.html
+setup(
+    name='DataProfiler',
     version=VERSION,
-
-    description='Data Profiler - What is in your data? ' 
-                'Detect statistics, schema, and '
-                'data specific information',
-    long_description=long_description,
+    description=DESCRIPTION,
+    # long_description=LONG_DESCRIPTION,
```

Comment:
Is this being removed or unintentionally left commented out?

Reply:
If `long_description` is staying, we should add:
```
long_description_content_type="text/markdown",
```

Reply:
I'll be making a seperate PR to update `long_description`, currently the markdown was failing and I don't know why. Would like to get this action in so we can push updates, long_description will likely come in tonight.